packages/protocol/contracts/L1/TaikoToken.sol

```diff
@@ -44,12 +44,12 @@ contract TaikoToken is EssentialContract, ERC20SnapshotUpgradeable, ERC20VotesUp
     /// @notice Burns tokens from the specified address.
     /// @param _from The address to burn tokens from.
     /// @param _amount The amount of tokens to burn.
-    function burn(address _from, uint256 _amount) public onlyOwner {
```

Comment:
There are other instances where we shall add 'payable' if this practice is applied. But prefer keeping checking if ether shall be accepted rather than save gas by removing it.

Reply:
@Brechtpd what's your take here?

Reply:
I think I also prefer not having functions marked as payable if they do not need to accept ETH, feels strangely inconsistent. Functions marked as `onlyOwner` are also likely not to be called often so the couple of gas saved doesn't seem to make much of a difference either.